README.md

```diff
@@ -0,0 +1,53 @@
+# CDS
```

Comment:
I think you should get Xing to review this README once she is back. I don't have much context on this anymore.